src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
+    window.open(`products/${searchItem}`)
+    setSearch("")
+  }
+
+  const handleKeyPress = (e) => {
+    if (e.key === 'Enter') {
+      handleClickSearch()
+    }
+  }
+
+  return (
+    <div className="home-search-box">
+      <input
+        value={searchItem}
+        type="text"
+        onChange={(e) => setSearch(e.target.value)}
```

Comment:
Since you decided to create a function and pass it here. Do you think it looks cleaner if this could be like other? 

```suggestion
        onChange={handleInputChange}
```

Reply:
I think this function has only one duty so I just put it like that without declaring a new function to handle it. it's just to set the state. What do you think?

Reply:
Actually, one function should have only one duty. In this case, its duty is to handle input change. Thus, creating a function for this will make it looks cleaner, isn't it?